index.html

```diff
@@ -354,10 +354,10 @@ <h2>
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </p>
         <div>
-            <form>
-                <input class="hover" placeholder="Full name">
-                <input class="hover" placeholder="Email address">
-                <textarea class="hover" placeholder="Enter text here"></textarea>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+                <input type="text" class="hover" maxlenght="30" placeholder="Full name" required>
```

Comment:
- [ ] Great job :+1: on using the `formspree site`  and post method to submit user pieces of information. But while using the formspree site, the information was not submitted successfully due to missed endpoint `{form_id}`. So please refer [formspree site](https://formspree.io/) how to use the endpoints for the action attribute on your form. The value of the endpoint for the action attribute example see below, but it's unique for every user.

***Example***
```
action="https://formspree.io/f/xrgrovrb"
```